packages/block-library/src/quote/deprecated.js

```diff
@@ -206,12 +290,14 @@ const v0 = {
 	migrate( attributes ) {
 		if ( ! isNaN( parseInt( attributes.style ) ) ) {
 			const { style, ...restAttributes } = attributes;
-			return migrateToQuoteV2( {
-				...restAttributes,
-			} );
+			return migrateTextAlign(
+				migrateToQuoteV2( {
+					...restAttributes,
+				} )
+			);
 		}
 
-		return migrateToQuoteV2( attributes );
+		return migrateTextAlign( migrateToQuoteV2( attributes ) );
```

Comment:
As noted earlier, this also needs fixing to accept the array returned by `migrateToQuoteV2`